app/styles/components/modules.scss

```diff
@@ -31,7 +31,12 @@ li.sortable-placeholder {
 	border: 1px dashed #CCC;
 	background: none;
 }
-
 .mtop20 {
 	margin-top: 20px;
 }
+.mtop15 {
+	margin-top: 15px;
+}
+.mtop5 {
+	margin-top: 5px;
+}

```

Comment:
@BorysekOndrej do jakého `scss` mám tohle dát, když to chci používat napříč celým webem?

Reply:
Nejsem si jistý, kam by to pěkně zapadalo. Vytvořil jsem pro to util, kam jsem to prozatím hodil.